src/eduid/userdb/identity.py

```diff
@@ -109,7 +109,12 @@ class CountryMixin:
 
     @property
     def country(self) -> Country:
-        return countries.get(self.country_code)
+        try:
+            country = countries.get(self.country_code)
```

Comment:
This will not raise a `KeyError`, `.get` will return `None` if the given key is not found.

This should either be

```py
country = countries.get(self.country_code)
if country is None:   # or: `if not country:` to include other cases, like empty string
   ...
```

or 

```py
try:
    country = countries[self.country_code]
except KeyError:
    ...
```

Reply:
Well, I'm assuming `countries` is a dict, but I now see this is part of `iso3166` package, which has a similar interface but different behaviour.

So I think that's OK 👍

Reply:
Yeah, got this in the logs :)
```
  File "/opt/eduid/sources/eduid/userdb/identity.py", line 112, in country
    return countries.get(self.country_code)
  File "/opt/eduid/webapp/lib/python3.9/site-packages/iso3166/__init__.py", line 505, in get
    raise KeyError(key)
KeyError: 'XB'
```